File: jcv-restassured-example/src/main/java/com/ekino/oss/jcv/example/jcvrestassuredexample/dto/OrderItemDto.java
/*
 * Copyright (c) 2019 ekino (https://www.ekino.com/)
 */
package com.ekino.oss.jcv.example.jcvrestassuredexample.dto;

import lombok.*;
import lombok.experimental.FieldDefaults;

import java.time.Instant;
import java.util.UUID;

@Builder
@NoArgsConstructor
@AllArgsConstructor
@Data
@EqualsAndHashCode
@FieldDefaults(level = AccessLevel.PRIVATE)
public class OrderItemDto {

    UUID id;
    String reference;
}

File: jcv-restassured-kotlin-example/src/main/kotlin/com/ekino/oss/jcv/example/jcvrestassuredexample/dto/OrderItemDto.kt
/*
 * Copyright (c) 2019 ekino (https://www.ekino.com/)
 */
package com.ekino.oss.jcv.example.jcvrestassuredexample.dto

import java.util.UUID

data class OrderItemDto(
    var id: UUID? = null,
    var reference: String? = null
)
